docs/xonshrc.rst

```diff
@@ -4,7 +4,7 @@ Xonsh allows you to have run control files to customize your shell behavior.  Th
 
 The system-wide ``xonshrc`` file controls options that are applied to all users of Xonsh on a given system.  You can create this file in ``/etc/xonshrc`` for Linux and OSX and in ``%ALLUSERSPROFILE%\xonsh\xonshrc`` on Windows.
 
-Xonsh also allows you to have a run control file in your home directory called ``~/.xonshrc``.  The options set in the local ``xonshrc`` only apply to the current user and will override any conflicting settings set in the system-wide control file.
+Xonsh also allows you to have a run control file in your home directory. It can either be directly in ``~/.xonshrc`` or for XDG compliance in ``~/.config/rc.xsh``. The options set in the local ``xonshrc`` only apply to the current user and will override any conflicting settings set in the system-wide control file.
```

Comment:
```suggestion
Xonsh also allows you to have a run control file in your home directory. It can either be in directly in the home directory at ``~/.xonshrc`` or for XDG compliance at ``~/.config/rc.xsh``. The options set in the local ``xonshrc`` only apply to the current user and will override any conflicting settings set in the system-wide control file.
```

Reply:
"in directly in"? Doesn't sound right to me.

Reply:
oops, you're right